backend/aurcache/src/builder/docker.rs

```diff
@@ -199,10 +216,7 @@ impl Builder {
             user: Some("ab".to_string()),
             cmd: Some(vec!["sh".to_string(), "-c".to_string(), cmd]),
             host_config: Some(HostConfig {
-                #[cfg(debug_assertions)]
-                auto_remove: Some(false),
-                #[cfg(not(debug_assertions))]
-                auto_remove: Some(true),
+                auto_remove: Some(auto_remove),
```

Comment:
Not directly related, I thought it might be cleaner to use the cfg to define a boolean and then assign it like a normal value, rather than having the field itself be conditional. I could move it out of this PR if desired.